spec/faraday/adapter_registry_spec.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+RSpec.describe Faraday::AdapterRegistry do
+  describe '.initialize' do
+    subject { described_class.new }
+
+    it { expect { subject.get(:FinFangFoom) }.to raise_error(NameError) }
```

Comment:
```suggestion
    it { expect { registry.get(:FinFangFoom) }.to raise_error(NameError) }
```

Reply:
The suggestion is: name the subject when we use it explicitly. 

Would that feel OK?

[Longer description](http://www.betterspecs.org/#subject)